<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyFirstQtAppClass</class>
 <widget class="QMainWindow" name="MyFirstQtAppClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyFirstQtApp</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>11</pointsize>
          <italic>false</italic>
          <bold>true</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 170, 0);
font: 700 11pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>PRIHLÁSENIE</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <fontweight>DemiBold</fontweight>
         </font>
        </property>
        <property name="text">
         <string>Vybrať typ používateľa:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="typPouzivatela">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <item>
         <property name="text">
          <string>Hosť</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Zamestnanec</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <fontweight>DemiBold</fontweight>
         </font>
        </property>
        <property name="text">
         <string>Bližšia špecifikácia užívateľa:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="typTypuPouzivatela">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <fontweight>DemiBold</fontweight>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Vybrať meno:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="menoPouzivatela">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <fontweight>DemiBold</fontweight>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Zadať heslo:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="heslo">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="prihlasenie">
        <property name="font">
         <font>
          <fontweight>DemiBold</fontweight>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(85, 170, 0);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Prihlásiť sa</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>330</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MyFirstQtApp.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>slotPreButon()</slot>
 </slots>
</ui>
